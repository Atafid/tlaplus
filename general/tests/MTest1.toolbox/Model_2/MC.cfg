\* CONSTANT definitions
CONSTANT
a <- const_12495091923297000
SPECIFICATION spec_12495091923298000
\* Generated at Wed Aug 05 14:53:12 PDT 2009