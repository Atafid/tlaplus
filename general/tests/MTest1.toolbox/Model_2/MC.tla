---- MODULE MC ----
EXTENDS MTest1, TLC

\* CONSTANT definitions
const_12495091923297000 == 
1
----

\* New definitions
foo == 2
----
\* Specification formula
spec_12495091923298000 ==
x=0 /\ [][x'=a]_x
----
====
\* Generated at Wed Aug 05 14:53:12 PDT 2009