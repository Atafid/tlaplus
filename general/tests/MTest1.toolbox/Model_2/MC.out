TLC2 Version 2.02 of 3 August 2009
@!@!@STARTMSG 2187:0 @!@!@
Running in Model-Checking mode.
@!@!@ENDMSG 2187 @!@!@
@!@!@STARTMSG 2220:0 @!@!@
Starting SANY...
@!@!@ENDMSG 2220 @!@!@
Parsing file MC.tla
Parsing file MTest1.tla
Parsing file C:\users\lamport\tla\tla2\org.lamport.tla.toolbox\StandardModules\TLC.tla
Parsing file C:\users\lamport\tla\tla2\org.lamport.tla.toolbox\StandardModules\Naturals.tla
Parsing file C:\users\lamport\tla\tla2\org.lamport.tla.toolbox\StandardModules\Sequences.tla
Semantic processing of module MTest1
Semantic processing of module Naturals
Semantic processing of module Sequences
Semantic processing of module TLC
Semantic processing of module MC
Semantic errors:

*** Errors: 1

line 10, col 1 to line 10, col 8 of module MC

Operator foo already defined or declared.


*** Warnings: 1

line 10, col 1 to line 10, col 8 of module MC

Multiple declarations or definitions for symbol foo.  This duplicates the one at line 10, col 1 to line 10, col 8 of module MTest1.  The original declaration or definition will be used, and this one will be ignored.



@!@!@STARTMSG 2219:0 @!@!@
SANY finished.
@!@!@ENDMSG 2219 @!@!@
@!@!@STARTMSG 1000:1 @!@!@
@!@!@STARTMSG 3002:0 @!@!@
Parsing or semantic analysis failed.
@!@!@ENDMSG 3002 @!@!@
@!@!@ENDMSG 1000 @!@!@
